tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,614 @@
+    ```
+    dig @ns1.dom.scw.cloud ingress.scw.your-domain.tld
+
+    ...
+    ;; ANSWER SECTION:
+    ingress.scw.your-domain.tld. 60 IN    A       195.154.68.56
+    ingress.scw.your-domain.tld. 60 IN    A       51.159.85.96
+    ...
+    ```
+
+Your multi-AZ Ingress Nginx Controller should now available on `http://ingress.scw.your-domain.tld`
+
+To automatically manage TLS certificates for your HTTPS ingresses, you can [deploy the cert-manager](https://cert-manager.io/).
+
+## Elasticsearch/Kibana as a stateful multi-AZ application
+
+In this last section, you will deploy a multi-AZ Elasticsearch and a Kibana instance to query it. It will allow you to work with the stateful constraints imposed by the use of persistent volumes in a multi-AZ architecture. As a prerequisite, you need to have a default `scw-bssd` storage class with a `volumeBindingMode` set to `WaitForFirstConsumer`. You can check your cluster with the `kubectl` command below. Previous clusters were provisioned with the binding mode set to `Immediate`, so if you are in this case you need to update your cluster to the latest patch version (kubernetes >=1.24.17, >=1.25.13, >=1.26.8, >=1.27.5 or >=1.28.1). The storage class will be automatically updated to the new binding mode during the update.
+    ```
+    kubectl get storageclasses.storage.k8s.io
+
+    NAME                 PROVISIONER        RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+    scw-bssd (default)   csi.scaleway.com   Delete          WaitForFirstConsumer   true                   131m
+    scw-bssd-retain      csi.scaleway.com   Retain          WaitForFirstConsumer   true                   131m
+    ```
+
+It is important to use a storage class with the `volumeBindingMode` option set to `WaitForFirstConsumer` when deploying a multi-AZ application with persistent volumes. 
+  This option ensures the volume is created only when the pod is scheduled on a node, addressing the specific AZ requirement. Creating the volume before scheduling the pod could result in the volume being placed randomly in an AZ, potentially causing attachment failures if the pod is scheduled in a different AZ. 
+  With the `WaitForFirstConsumer` option, volumes are created on the same AZ as the node where the pod is scheduled, effectively spreading volumes across different AZs as pods are distributed.
+
+1. For this new section, create a new Terraform file `elasticsearch.tf`. Deploying a clustered application commonly implies a more complex deployment logic. For Elasticsearch and Kibana, this logic is managed by ECK, the Elasticsearch operator for Kubernetes. It will be in charge of creating Elasticsearch nodes and configuring them to make a cluster. It will also automatically configure Kibana to connect to the Elasticsearch cluster. So the first thing to do is to deploy the ECK operator using its chart with the Helm provider.
```

Comment:
```suggestion
1. Ccreate a new Terraform file `elasticsearch.tf` for this section. Deploying a clustered application commonly implies a more complex deployment logic. For Elasticsearch and Kibana, this logic is managed by ECK, the Elasticsearch operator for Kubernetes. It will be in charge of creating Elasticsearch nodes and configuring them to make a cluster. It will also automatically configure Kibana to connect to the Elasticsearch cluster. So the first thing to do is to deploy the ECK operator using its chart with the Helm provider.
```